plugins/postcss-oklab-function/test/variables.expect.css

```diff
@@ -1,18 +1,18 @@
 :root {
-	--firebrick: rgb(90, 66, 47);
-	--firebrick-a50: rgba(83, 67, 71, 0.5);
+	--firebrick: rgb(141, 0, 58);
+	--firebrick-a50: rgba(139, 0, 68, 0.5);
```

Comment:
With the optimized output these tests now require out of gamut colors to fully test the generated CSS.